product_standard_margin/readme/ROADMAP.rst

```diff
@@ -0,0 +1,7 @@
+* This module will not work properly if used in a multicompany context with product
+  prices depending on the company.
+
+* It should be great to use the odoo widget ``percentpie`` for the display of the
+  field ``standard_margin_rate`` but for the time being, the field is not displayed
+  properly.
+  Ref of the Odoo Issue: https://github.com/odoo/odoo/issues/42869
```

Comment:
Did you tested if this bug was solved ?

Reply:
done

Reply:
Hi. Just tested, it doesn't seems to work. Or did I missed something ?

Reply:
![image](https://user-images.githubusercontent.com/3407482/161710557-3abcd7fb-9c15-4186-9c52-7719f1b3f314.png)

Reply:
Well, the bug in 14.0 still remains. However it's a pity to not use this great widget. 

Analysing : 
A text is displayed after the percent graph, because of : 
https://github.com/odoo/odoo/blob/14.0/addons/web/static/src/xml/base.xml#L1370

This line is defined because they are used in a widget that should be defined as ``statinfo``, but both are incompatible.

https://github.com/odoo/odoo/blob/14.0/addons/mass_mailing/views/mailing_mailing_views.xml#L125

Not sure that it will be fixed in odo,o because this widget is used only in one module, in one form. (``mass_mailing``)

**Option 1**
- conserve the use of ``percentpie`` widget
- keep an updated text to say that there is a bug to be fixed in odoo. (that can be fixed, removing a line in the ``base.xml`` file).

**Option 2**
- remove the use of ``percentpie`` widget.

What do you think ?

Reply:
Option 1 : we have customers who like the percentpie ;-)